.github/workflows/check-semver.yml

```diff
@@ -0,0 +1,23 @@
+name: Check Semver
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Is there a benefit of also running that on branch main? Probably not as its only relevant when doing the push then. So I think it's currently the only relevant target? Making sure a breaking PR is getting detected?

Reply:
I think it will be sufficient to run it for PRs for now. There is https://github.com/obi1kenobi/cargo-semver-checks-action/pull/65 which will make things easier in the future.